### PR TITLE
Bug fix: Update "on.js" documentation comment

### PR DESCRIPTION
**Pull request by briandavidclark**

### CODE REVIEW

**Comment by customcommander** [comment]
Thanks @briandavidclark . I'm leaving this open for 24h just to give other maintainers and contributors to say something but I don't think anybody will object to this anyway ;) Will merge tomorrow if somebody doesn't do it sooner.

**Review by adispring — APPROVED** [review]

**Pull request merged**
